image-scanner.sh

```diff
@@ -54,17 +135,20 @@ for image in $REPO_LIST; do
     image_name=$(echo "$image" | tr '/' '-')
 
     # Define the Output File
-    output_file_txt="$ROOT_DIR/${DIR}/${image}-${TODAY_DATE}.html"
+    output_file_txt="$ROOT_DIR/${DIR}/${image}.html"
 
     # Run Trivy scan on the image
     echo "Scanning image: $image"
     trivy image --severity HIGH,CRITICAL --format template --template "@html.tpl" --output "$output_file_txt" "${ORG}/$image_name"
```

Comment:
This will increase the scanning time, since the scanning is done once for HTML and again for text.

Reply:
Yes, agreed. But currently trivy does not support output in multiple formats. To see details about vulnerabilities we can use html report format, for summary we can use the summary file